ui/components/src/ThresholdsEditor/ThresholdInput.tsx

```diff
@@ -47,7 +47,7 @@ export function ThresholdInput({
         key={key}
         inputRef={inputRef}
         type="number"
-        value={value === 0 ? undefined : value}
+        value={value === 0 ? '' : value}
```

Comment:
Setting the input to `undefined` makes React think it's an uncontrolled component. It then throws an error about switching from uncontrolled to controlled. Instead falling back to an empty string, which should accomplish the same UX without the react error.